lib/utils.pm

```diff
@@ -816,23 +817,46 @@ sub fully_patch_system {
 Connect to the remote host C<$host> using ssh and update the system by
 running C<zypper patch> twice. The first run will update the package manager,
 the second run will update the system.
+Transactional systems like SLE micro shall use C<transactional_update patch> and reboot, instead, 
 
 =cut
 
 sub ssh_fully_patch_system {
     my $remote = shift;
+    my $cmd = "ssh $remote 'sudo zypper -n patch --with-interactive -l'";
+    my $cmd_name = "zypper patch";
```

Comment:
I find those two variables quite confusing.

Reply:
This module full reverted to master version, changes moved in publicclud/utils. See last commit